.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
есть возможности оптимизировать код

Reply:
внимательней посмотрите что и как используется

Reply:
Added some optimization to function that change marker icon. I moved it to the map.service,  because it double used in my code, I had two equal functions before that  (